### PR TITLE
Helppage/nick

### PR DESCRIPTION
**Pull request by Nick-Ohman**
# Description

Added routing and links into homescreen as well as added a few working tests in a test file, build out small help component and added a test file wit ha couple tests in that.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by bramirez96 — DISMISSED** [review]
Looks great!

**Review by SandraCoburn — DISMISSED** [review]
Good looking tests! Help page looks great.

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+//import { createStore, applyMiddleware } from 'redux';
```
Comment (Schrese) [inline src/index.js:3]: Thanks for getting rid of these, I completely forgot to do that

Comment (Schrese) [inline src/index.js]: We might want to take out the commented our sections in this file

**Review by Schrese — APPROVED** [review]
Thanks. Looks great!

**Review by SandraCoburn — APPROVED** [review]
Looks good!

**Review by bramirez96 — APPROVED** [review]
Looks great!

File: src/components/pages/HomeScreen/HomeScreen.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ function HomeScreen () {
               mode="inline"
               defaultSelectedKeys={['dashboard']}
             >
-              <Menu.Item key="dashboard"></Menu.Item>
+              <Menu.Item key="dashboard"><Link to="/dashboard"></Link>Dashboard</Menu.Item>
```
Comment (SandraCoburn) [inline src/components/pages/HomeScreen/HomeScreen.js]: Could you change this Link to "/parent-dashboard"?

**Pull request merged**